routes/usersRouter.js

```diff
@@ -16,4 +19,40 @@ userRouter.post(
   users.verifyUser
 );
 
+userRouter.post(
+  "/login",
+  validate(validation.loginSchema),
+  function (req, res, next) {
+    passport.authenticate("local-login", function (err, user, info) {
+      if (err) {
+        logger.error("An error has occured: ", err);
+        return next(err);
+      }
+      if (!user) {
+        return res
+          .status(401)
+          .send({ msg: "Authentication failed!, wrong email or password " });
+      }
+      req.login(user, { session: false }, (err) => {
+        if (err) {
+          logger.error("An error has occured: ", err);
```

Comment:
Since you are referencing this statement **An error has occurred:** multiple times, declare it as a constant somewhere and just reference though the variable name.